File: Common/CommonEntities/QuestionAnswer/UserQuestionAnswer.cs
using CommonEntities.Base;
using static CommonEntities.Enums;

namespace CommonEntities.QuestionAnswer
{
    public class UserQuestionAnswer : BaseRequest
    {
        public int id { get; set; }
        public DateTime? CreatedDate { get; set; }
        public DateTime? UpdatedDate { get; set; }
        public string Question { get; set; }
        public string Answer { get; set; }
        public int? DomainId { get; set; }
        public int? UserId { get; set; }
        public QuestionAnswerStatus QuestionAnswerStatus { get; set; }
        public string UseFullLink { get; set; }
        public int? PrepairedRatio { get; set; }
        public int? SystemQuestionAnswerId { get; set; }
    }
}

File: UI/MyInterviewPreparation/ClientApp/src/Entities/UserQuestionAnswer.ts
import { QuestionAnswerStatus } from "./Enums";
import { BaseRequest } from "./BaseRequest";

export interface UserQuestionAnswer extends BaseRequest{
    id: number;
    createdDate: string | null;
    updatedDate: string | null;
    question: string ;
    answer: string ;
    domainId: number ;
    userId: number ;
    questionAnswerStatus: QuestionAnswerStatus;
    useFullLink: string;
    prepairedRatio: number | null;
    systemQuestionAnswerId: number | null;
}